src/main/java/com/stuypulse/robot/subsystems/arm/Arm.java

```diff
@@ -93,6 +117,7 @@ public final void setWristTargetState(Rotation2d wrist) {
 
     public final void setTrajectory(ArmBFSField trajectory) {
         this.trajectory = Optional.ofNullable(trajectory);
+        voltages = Optional.empty();
```

Comment:
The intake button is bound to a trajectory which disables voltage control.

Reply:
Alternative: If voltage control is already enabled then don't default to trajectory.